docs/api/python/frozen/pyopencolorio_gradingprimarytransform.rst

```diff
@@ -23,7 +23,7 @@
 
       Overloaded function.
 
-      1. __init__(self: PyOpenColorIO.GradingPrimaryTransform, style: PyOpenColorIO.GradingStyle = <GradingStyle.GRADING_LOG: 0>, values: PyOpenColorIO.GradingPrimary, dynamic: bool = False, dir: PyOpenColorIO.TransformDirection = <TransformDirection.TRANSFORM_DIR_FORWARD: 0>) -> None
+      1. __init__(self: PyOpenColorIO.GradingPrimaryTransform, style: PyOpenColorIO.GradingStyle = <GradingStyle.GRADING_LOG: 0>, values: OpenColorIO_v2_0rc1::GradingPrimary, dynamic: bool = False, dir: PyOpenColorIO.TransformDirection = <TransformDirection.TRANSFORM_DIR_FORWARD: 0>) -> None
```

Comment:
I'll do some investigation soon into why C++ types are slipping into the frozen Python docs.

Reply:
I'll revert this and will review rst files